### ISSUE TITLE
Compatibility with UISP Fiber 4.6.0?

### ISSUE THREAD

**Issue by cdalvaro** [Unifi-Tools/UFiber.Configurator#26]
Has anyone tried to update to [UISP Fiber 4.6.0](https://community.ui.com/releases/UISP-Fiber-4-6-0/912f90cb-ac94-49e8-96ac-5cab59298b3e) version?

If so, how was the process? There was any issue after the update?

Thank you in advance!

**Comment by micallan17** [comment]
I've updated to 4.6.0 but since 4.4.2 it seems we're unable to change the mac address as it reverts back to the original hardcoded mac address. If I downgrade to anything below 4.4.2, it brings back the modified mac address and it works with my ISP who authenticates with serial/vender + mac address. If your ISP only requires the serial to authenticate then it still can be changed on 4.4.2+ firmwares it seems from my testing atleast.

**Comment by pmiron** [comment]
> I've updated to 4.6.0 but since 4.4.2 it seems we're unable to change the mac address as it reverts back to the original hardcoded mac address. If I downgrade to anything below 4.4.2, it brings back the modified mac address and it works with my ISP who authenticates with serial/vender + mac address. If your ISP only requires the serial to authenticate then it still can be changed on 4.4.2+ firmwares it seems from my testing atleast.

Thanks for the reply
I guess you run the config tool after the update to 4.6.0, right?

**Comment by micallan17** [comment]
if you've changed the serial prior to 4.6.0, I don't think you need run the tool again since the serial does seem to persist from what I can tell. I mean, the only way you'll find out is by updating and once it reboots check if it connects to your ISP. The one that I see doesn't persist after updating to anything above 4.4.x, is the MAC and so far I haven't been able to write a custom mac on anything above 4.4.x.

**Comment by pmiron** [comment]
I saw problems changing the Profile(from 4 to 2 for instance) then the SN is changed to default. All that with version 4.3.1.
So I had to run the tool again. That's why I'm asking if the tool was rerun after the update

**Comment by jspavlick** [comment]
I have encountered this with my UFiber Loco. 4.3.1 works, and takes the flashed MAC address, but 4.4.2 and 4.4.6 do *not*, and they revert to the hardware MAC address.

**Comment by hiagocosta** [comment]
mine on 4.4.6 keeps the changed value, I want to try a newer version, i'm facing some low speed, and maybe a newer can improve this

**Comment by jspavlick** [comment]
There is no newer version. Ubiquiti seems to have stopped supporting this device, or at least stopped actively working on improving it.

**Comment by hiagocosta** [comment]
yes, they are still updating, just download [UF OLT, UF-OLT4](https://dl.ui.com/firmwares/ufiber/v4.9.0/UF-OLT.v4.9.0.6363.241129.1117.tar) and extract, in the file squashfs.tmp change the extension to .rar, open with 7zip and check folder below, after they change the product line name, they changed how they update

![Image](https://github.com/user-attachments/assets/548978dd-03c1-483d-b087-bc3281a0ee9b)

**Comment by micallan17** [comment]
Am on 4.9.0 and I had updated from 4.8.0 and the cloned sn persisted the update. I no longer need to clone the mac address since I have the ufiber loco in bridge mode, so the MAC address is cloned on the unifi gateway ultra interface and so I don’t have to use the fiber loco for routing. The cloned sn on the ufiber loco is only for fiber authentication when set in bridge mode, I use profile 2 (Huawei olt) if that helps.

**Comment by hiagocosta** [comment]
> Am on 4.9.0 and I had updated from 4.8.0 and the cloned sn persisted the update. I no longer need to clone the mac address since I have the ufiber loco in bridge mode, so the MAC address is cloned on the unifi gateway ultra interface and so I don’t have to use the fiber loco for routing. The cloned sn on the ufiber loco is only for fiber authentication when set in bridge mode, I use profile 2 (Huawei olt) if that helps.

did the tool keeps the cloned MAC? have you tried? I need MAC, and SLID to be cloned for my ISP

**Comment by jspavlick** [comment]
Wow interesting! Where I am right now my OLT is an ALCU so there is no firmware profile for them, so I can’t test 4.9.0 right now.

**Comment by hiagocosta** [comment]
does anyone know if the firmware update can be downgraded? I can try if the newer one is compatible.

I downloaded all the next versions after 4.4.6 to test if still works, but I'm afraid if I cannot downgrade it if something goes wrong

**Comment by micallan17** [comment]
> > Am on 4.9.0 and I had updated from 4.8.0 and the cloned sn persisted the update. I no longer need to clone the mac address since I have the ufiber loco in bridge mode, so the MAC address is cloned on the unifi gateway ultra interface and so I don’t have to use the fiber loco for routing. The cloned sn on the ufiber loco is only for fiber authentication when set in bridge mode, I use profile 2 (Huawei olt) if that helps.
> 
> did the tool keeps the cloned MAC? have you tried? I need MAC, and SLID to be cloned for my ISP

With the Mac address am not sure, as I mentioned I didn’t need to clone the mac address on the Ufiber loco as I clone it on the unifi gateway instead. My isp just uses the sn for authentication it seems and the service is locked to the mac address which is why I clone it on the unifi gateway.

**Comment by micallan17** [comment]
> Wow interesting! Where I am right now my OLT is an ALCU so there is no firmware profile for them, so I can’t test 4.9.0 right now.

Yeah the Ufiber Loco only supports 4 profiles:

Profile 1: UI OLT
Profile 2: Huawei OLT
Profile 3: FiberHome OLT
Profile 4: ZTE OLT

**Comment by micallan17** [comment]
> does anyone know if the firmware update can be downgraded? I can try if the newer one is compatible.
> 
> I downloaded all the next versions after 4.4.6 to test if still works, but I'm afraid if I cannot downgrade it if something goes wrong

It should work for downgrades i have a spare ufiber loco which i can test with and report back

**Comment by micallan17** [comment]
> does anyone know if the firmware update can be downgraded? I can try if the newer one is compatible.
> 
> I downloaded all the next versions after 4.4.6 to test if still works, but I'm afraid if I cannot downgrade it if something goes wrong

downgrades work:

from 4.9.0

![Image](https://github.com/user-attachments/assets/c6cebf5b-7900-417c-9872-f2301849f43e)

to 2.0.1

![Image](https://github.com/user-attachments/assets/a02ee3be-d0e6-4412-859c-3469c9fb208f)

**Comment by hiagocosta** [comment]
> > does anyone know if the firmware update can be downgraded? I can try if the newer one is compatible.
> > I downloaded all the next versions after 4.4.6 to test if still works, but I'm afraid if I cannot downgrade it if something goes wrong
> 
> downgrades work:
> 
> from 4.9.0
> 
> ![Image](https://github.com/user-attachments/assets/c6cebf5b-7900-417c-9872-f2301849f43e)
> 
> to 2.0.1
> 
> ![Image](https://github.com/user-attachments/assets/a02ee3be-d0e6-4412-859c-3469c9fb208f)

wow,  2.0.1 is a hell of a downgrade lol
on the UI support, is told that the system cannot downgrade below 4.0.1, but now I checked is only to OLT
i'll try today changig the version, one by one to check when it (could) stop to work.

**Comment by hiagocosta** [comment]
Hi guys, after a period of time (vacation) I now tested the ufiber with pathed configurations and here is my report:

 (i didn't try to run ufiber configurator, only with 4.4.6 that I did and it worked until now)

download the 4.6.0 version and update from 4.4.6 and test it -> OK
download the 4.8.0 version and update the firmware and it keeps me connected -> OK

so by this I can conclude that ufiber configurator if it patched on 4.4.6 will keep the patched SLID, S/N until 4.8.0 version, I didn't tried the last 4.9.0 for now but in the future maybe I will

As I saw, the new 4.9.0 was upgraded linux version to bookworm, it is a good and sensible point

**Comment by micallan17** [comment]
Am currently on 4.9.0 and it kept the SN after the upgrade, mine is set up in Bridge mode so no meed to clone the MAC address on my end.Sent from my iPhoneOn 09-May-2025, at 10:34 AM, Hc ***@***.***> wrote:﻿
hiagocosta left a comment (Unifi-Tools/UFiber.Configurator#26)
Hi guys, after a period of time (vacation) I now tested the ufiber with pathed configurations and here is my report:
(i didn't try to run ufiber configurator, only with 4.4.6 that I did and it worked until now)
download the 4.6.0 version and update from 4.4.6 and test it -> OK
download the 4.8.0 version and update the firmware and it keeps me connected -> OK
so by this I can conclude that ufiber configurator if it patched on 4.4.6 will keep the patched SLID, S/N until 4.8.0 version, I didn't tried the last 4.9.0 for now but in the future maybe I will
As I saw, the new 4.9.0 was upgraded linux version to bookworm, it is a good and sensible point
